source/ssh.rst

```diff
@@ -0,0 +1,267 @@
+.. _ssh:
+
+SSH
+===
+
+Quickstart
+----------
+
+Do this on *each* machine you will be connecting from:
+
+- Create a new keypair::
+
+    ssh-keygen -t ed25519 -C "your_email@example.com"
+
+- Copy the public key to each host you want to connect to:
+
+  .. code-block:: bash
+
+      # for each host you want to connect to
+      ssh-copy-id user@hostname
+
+- Log in to your GitHub account, GitLab account, or any other services that use
+  SSH keys. Paste the contents of ``~/.ssh/id_ed25519.pub`` into the settings
+  of your account profile. 
+
+- Once per bash session, run this to load your key into memory for the duration
+  of that session:
+
+  .. code-block:: bash
+
+      # See below for more info on macOS
+      eval `ssh-agent`; ssh-add
+
+
+More details below....
+
+Overview
```

Comment:
should overview be above quick start?

Reply:
I was kind of torn about the ordering, but since you bring it up that was the tie-breaker -- I changed it to put the overview first